<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>370</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>520</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>250</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="toolTipDuration">
            <number>0</number>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="columnCount">
            <number>0</number>
           </property>
          </widget>
          <widget class="QTabWidget" name="tabWidget01">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="tabWidgetPage1">
            <attribute name="title">
             <string>属性</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tabWidgetLog">
            <attribute name="title">
             <string>日志</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tabWidgetPage2">
            <attribute name="title">
             <string>空间运动</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tabWidgetPage3">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <attribute name="title">
             <string>关节运动</string>
            </attribute>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="occWidget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>26</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>导入</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionRbtImport"/>
     <addaction name="separator"/>
     <addaction name="actionWopImport"/>
     <addaction name="separator"/>
     <addaction name="actionRbtJointImport"/>
     <addaction name="separator"/>
     <addaction name="actionToolImport"/>
     <addaction name="separator"/>
     <addaction name="actionSTLImport"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSet">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionSoftWareHelp"/>
    <addaction name="separator"/>
    <addaction name="actionDeveloperState"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>日志</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSet"/>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExport">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionRbtImport">
   <property name="text">
    <string>导入机器人</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionWopImport">
   <property name="text">
    <string>导入工件</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRbtJointImport">
   <property name="text">
    <string>导入机器人关节</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionToolImport">
   <property name="text">
    <string>导入工具</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSTLImport">
   <property name="text">
    <string>导入点云</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSoftWareHelp">
   <property name="text">
    <string>软件使用说明</string>
   </property>
  </action>
  <action name="actionDeveloperState">
   <property name="text">
    <string>开发者声明</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>碰撞检测开启/关闭</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>基础视图</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>高级视图</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>偏好设置</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>位姿变换</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
